### PR TITLE
Remove system root from DBus methods

### PR DESCRIPTION
**Pull request by poncovka**
Remove the argument for the system root from the DBus methods that
return installation and configuration tasks. The path to the system
root doesn't change anymore.

Replace functions getSysroot, setSysroot and getTargetPhysicalRoot.

**Depends on:**
https://github.com/rhinstaller/initial-setup/pull/73
https://github.com/daveyoung/kdump-anaconda-addon/pull/23

### CODE REVIEW

File: pyanaconda/core/util.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def augmentEnv():
     return env
 
 
-def setSysroot(path):
+def set_system_root(path):
```
Comment (jkonecny12) [inline pyanaconda/core/util.py]: Don't you want to rename this to `mount_system_root` or something similar. You are not setting path or anything now.

Comment (poncovka) [reply pyanaconda/core/util.py]: I was thinking about it, but from the "user" point of view, you just want to tell Anaconda what is the actual path to the system root. The bind mount behind it is just an implementation detail, that could change again in the future. The name `mount_system_root` suggests that the real system root will be mounted somewhere and that is not correct.

Comment (jkonecny12) [reply pyanaconda/core/util.py]: Still looks to me like a giant side effect. If we don't want to rename the method at least document that mount will happen and why it is happening.

Comment (poncovka) [reply pyanaconda/core/util.py]: Fixed.

**Comment by poncovka** [comment]
Rebased.

**Pull request merged**